examples/transform.js

```diff
@@ -1,6 +1,6 @@
 var write = require('csv-write-stream')
-var through = require('through2')
```

Comment:
there's value in showing how to use this with `through2` since that's still widely used. renaming the original `transform.js` to `transform-through2.js` and then modifying this file would've been a better route.